File: mbrs/src/main/java/com/ftn/mbrs/service/StavkaCenovnikaService.java
//Fri May 11 23:51:24 CEST 2018
//Generisano na osnovu sablona: service.ftl
//Element modela: GremoNaElektriko::StavkaCenovnika


package com.ftn.mbrs.service;

import java.util.List;

import com.ftn.mbrs.model.StavkaCenovnika;

public interface StavkaCenovnikaService {

	public StavkaCenovnika save(StavkaCenovnika stavkaCenovnika, Long tipPrikljuckaId, Long cenovnikId);
	
	public StavkaCenovnika update(StavkaCenovnika stavkaCenovnika);

	public String delete(Long id);
	
	public StavkaCenovnika findOne(Long id);
	
	public List<StavkaCenovnika> findAll();
}

File: mbrs/src/main/resources/static/js/services/StavkaCenovnikaService.js
//Sat May 12 11:32:37 CEST 2018
//Generisano na osnovu sablona: jsservice.ftl
//Element modela: GremoNaElektriko::StavkaCenovnika

angular.module('gremoNaElektrikoApp.StavkaCenovnikaService', [])
	.factory('StavkaCenovnikaService', function($http) {
		var factory = {};
		
		factory.findAll = function() {
			return $http.get('/gremoNaElektriko/stavkaCenovnika');
		}
		
		factory.createStavkaCenovnika = function(stavkaCenovnika , tipPrikljuckaId, cenovnikId) {
			return $http.post('/gremoNaElektriko/stavkaCenovnika'+'/'+tipPrikljuckaId+'/'+cenovnikId, stavkaCenovnika);
		}

		factory.deleteStavkaCenovnika = function(id) {
			return $http.delete('/gremoNaElektriko/stavkaCenovnika/'+id);
		}
		
		factory.updateStavkaCenovnika = function(stavkaCenovnika) {
			return $http.put('/gremoNaElektriko/stavkaCenovnika', stavkaCenovnika);
		}
		
		return factory;
	});